File: IntegrationGuideForAndroid/Apps/clientJava/src/main/java/com/example/integrationguide/Application.java
// Copyright 2023 VMware, Inc.
// SPDX-License-Identifier: BSD-2-Clause

package com.example.integrationguide;

import androidx.annotation.NonNull;

import com.airwatch.event.SDKClientConfig;
import com.airwatch.event.WS1AnchorEvents;

public class Application extends android.app.Application implements SDKClientConfig {
    @NonNull
    @Override
    public WS1AnchorEvents getEventHandler() {
        return new AppWS1AnchorEvents();
    }
}

File: IntegrationGuideForAndroid/Apps/clientKotlin/src/main/java/com/example/integrationguide/Application.kt
// Copyright 2023 VMware, Inc.
// SPDX-License-Identifier: BSD-2-Clause

package com.example.integrationguide

import com.airwatch.event.SDKClientConfig
import com.airwatch.event.WS1AnchorEvents

class Application: android.app.Application(), SDKClientConfig {

    override fun getEventHandler(): WS1AnchorEvents {
        return AppWS1AnchorEvents()
    }

}